web/packages/teleterm/src/mainProcess/mainProcess.ts

```diff
@@ -187,9 +201,7 @@ export default class MainProcess {
    * create might throw an error if spawning a child process fails, see initTshd for more details.
    */
   static create(opts: Options) {
-    const instance = new MainProcess(opts);
-    instance.init();
-    return instance;
+    return new MainProcess(opts);
```

Comment:
We might as well get rid of `create` completely at this point if it's just a thin wrapper for the constructor.